pkgs/development/python-modules/pick-colour-picker/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pygobject3
+, pycairo
+, glib
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+buildPythonPackage rec {
+  pname = "pick-colour-picker";
+  version = "1.5.0-3ec940";
```

Comment:
Should be `unstable-$commitDate` according to packaging guidelines.

Reply:
Which date format?

Reply:
There is only one format [ISO 8601](https://xkcd.com/1179/) 😉